pages/dkp/kommander/2.0/install/air-gapped/index.md

```diff
@@ -36,7 +36,7 @@ export VERSION=v2.0.0
 1.  Download the `kommander_image_bundle_v2.0.0_linux_amd64.tar` file.
 
     ```bash
-    wget "https://downloads.d2iq.com/kommander/airgapped/${VERSION}/kommander_image_bundle_${VERSION}_linux_amd64.tar" | tar -xvf -
+    wget "https://downloads.d2iq.com/kommander/airgapped/${VERSION}/kommander_image_bundle_${VERSION}_linux_amd64.tar" -O - | tar -xvf -
```

Comment:
I tried this - and it doesn't work. We should make sure we have the right image bundle for this - is this actually available?

```suggestion
    wget "https://downloads.d2iq.com/kommander/airgapped/${VERSION}/kommander_image_bundle_${VERSION}_linux_amd64.tar" -O - | tar -xvf -
```

Reply:
Is airgap actually available like this @takirala @karishmagadiya ?